internal/version/version.go

```diff
@@ -0,0 +1,51 @@
+package version
+
+import (
+	"fmt"
+	"regexp"
+	"runtime/debug"
+	"strings"
+)
+
+// Variables sent at link time via ldflags
+var (
+	Version   string
+	GitCommit string
+	Label     string
+)
+
+// Get returns the current CLI version, preferring the value set via ldflags
+func Get() string {
+	if Version != "" || GitCommit != "" {
+		return getVersionViaLdFlags()
+	}
+
+	info, ok := debug.ReadBuildInfo()
+	if ok {
+		versionParts := strings.Split(info.Main.Version, "-")
```

Comment:
I think that for something such as  
`go install github.com/ionos-cloud/ionosctl/v6@v6.7.2`
`go: downloading github.com/ionos-cloud/ionosctl/v6 v6.7.2`

the slice `versionParts` will get `["v6.7.2"]` value which will end up reporting an `unknown` version possibly.

Reply:
Yeah, that might be the case, thanks! We can add an early return case to see if `info.Main.Version` is a semantic tag, in which case we could return it directly